ui/app/components/file-to-array-buffer.js

```diff
@@ -0,0 +1,57 @@
+import Component from '@ember/component';
+import filesize from 'filesize';
+
```

Comment:
i saw that you added the `FormSaveButtons` -- should this component be added to Storybook too?  even if you decide not to add it, could you add JSDoc comments here so we know what the component is for / what the properties are, etc?

Reply:
Yeah there are JS doc comments (the old style), but I should convert this and add it to Storybook.

Reply:
Updated and added to storybook in https://github.com/hashicorp/vault/pull/7410/commits/66123c3ee4655f64728db74301ce39e17866f943 !